backend/worker/worker.go

```diff
@@ -1109,6 +1110,24 @@ func (w *Worker) StartLogAlertWatcher(ctx context.Context) {
 	log_alerts.WatchLogAlerts(ctx, w.Resolver.DB, w.Resolver.MailClient, w.Resolver.RH, w.Resolver.Redis, w.Resolver.ClickhouseClient, w.Resolver.LambdaClient)
 }
 
+func (w *Worker) StartSessionDeleteJob(ctx context.Context) {
+	retentionEnv := os.Getenv("SESSION_RETENTION_DAYS")
+	if retentionEnv == "" {
+		log.WithContext(ctx).Info("SESSION_RETENTION_DAYS not set, skipping SessionDeleteJob")
+		return
+	}
+	sessionRetentionDays, err := strconv.Atoi(os.Getenv("SESSION_RETENTION_DAYS"))
+	if err != nil {
+		log.WithContext(ctx).Error("Error parsing SESSION_RETENTION_DAYS, skipping SessionDeleteJob")
+		return
+	}
+	if sessionRetentionDays <= 0 {
+		log.WithContext(ctx).Error("sessionRetentionDays <= 0, skipping SessionDeleteJob")
+		return
+	}
```

Comment:
Do we have any sort of alerting for job run failures? Just wondering if these errors could get lost in the noise since they run kind of infrequently.

Reply:
agree it's probably worth some better visibility for this and other jobs (e.g. alert evaluation) - ticketing to look into this as part of the alerts work